nullaway/src/main/java/com/uber/nullaway/handlers/CompositeHandler.java

```diff
@@ -202,7 +205,10 @@ public NullnessHint onDataflowVisitFieldAccess(
     for (Handler h : handlers) {
       NullnessHint n =
           h.onDataflowVisitFieldAccess(node, symbol, types, context, apContext, inputs, updates);
-      nullnessHint = nullnessHint.merge(n);
+      nullnessHint = nullnessHint.mergeByPriority(n);
+      if (nullnessHint.isHighestPriority()) {
+        break;
```

Comment:
Again, this is unsafe, we don't care only about the return value from this method, we care that the method is called for every handler that implements it.

Reply:
removed